Dockerfile

```diff
@@ -57,6 +57,8 @@ COPY --from=builder /celestia-app/build/celestia-appd /bin/celestia-appd
 COPY --chown=${USER_NAME}:${USER_NAME} docker/entrypoint.sh /opt/entrypoint.sh
 # Set the user to celestia.
 USER ${USER_NAME}
+# Set the working directory to the home directory.
+WORKDIR ${CELESTIA_HOME}
```

Comment:
> This should be a non-breaking change.

Won't this change be breaking to scripts that assume the working directory is `/`? In other words, scripts that were written against the previous Docker container _may_ break.

> If the WORKDIR instruction is not specified in a Dockerfile, the default WORKDIR is the root directory (/).

Ref: https://www.baeldung.com/ops/docker-default-workdir

Reply:
Thanks for bringing this up. I checked out scripts, and we always use absolute paths.

I'm not sure if externals use the container with custom scripts as well.

Reply:
Also, @cmwaters uses absolute paths in the e2e tests using knuu